### PR TITLE
doc: Add Transport @macfja/pino-fingers-crossed

### PR DESCRIPTION
**Pull request by MacFJA**
Add a new transport (`@macfja/pino-fingers-crossed`) in the documentation.

### CODE REVIEW

File: docs/transports.md [hunk 1]
```diff
@@ -1119,6 +1120,29 @@ logger.error(new Error("error"));
...
 ```
 
+<a id="macfja-pino-fingers-crossed"></a>
+### @macfja/pino-fingers-crossed
+
+[@macfja/pino-fingers-crossed](https://github.com/MacFJA/js-pino-fingers-crossed) is a Pino v7+ transport that holds logs until a log level is reached, allowing to only have logs when it matters.
+
+```js
+import pino from "pino"
+import fingersCrossed, { enable } from "@macfja/pino-fingers-crossed"
```
Comment (jsumners) [inline docs/transports.md:1130]: At the moment, my only feedback is to follow the same style as the other examples. That is, not ESM.

Comment (MacFJA) [reply docs/transports.md:1130]: My bad, I thought that both were okay.

Comment (jsumners) [reply docs/transports.md:1130]: I'm just looking for consistency across the docs.

Comment (MacFJA) [reply docs/transports.md]: Updated!

_(Sorry for the delay, I have to make my lib compatible with commonjs first)_

**Review by jsumners — APPROVED** [review]

**Review by mcollina — APPROVED** [review]
lgtm

**Pull request merged**